Server/image_functions/image_module/contrast_stretching.py

```diff
@@ -0,0 +1,53 @@
+def contrast_stretching(id1, id2):
+    """
+        Process the input image with contrast_stretching
+        and check the image size
+
+        :param id1: the input should be the name string of the image
+        :param id2: the input should be the name string of the processed image
+        :raises ImportError:  if input is not a string
+        :raises NameError:: if the id1 image file does not exist
+        :raises AssertionError: if the image array does not contain
+                three layers
+
+        :returns: a python dictionary contains the base64 string
+                  of the processed image and the image size
+        :rtype: python dictionary
+        """
+    try:
+        import PIL
+        from PIL import Image
+        import numpy as np
+        from skimage import exposure
+        import logging
```

Comment:
is `logging` in `requirements.txt`? Make sure you have done `git pull origin master` to update your branch with Yi's code

Reply:
It is not in the requirements.txt. There is some online resources said that logging is a part of the Python standard library, so we do not need to pip install logging. 
Reference: https://stackoverflow.com/questions/33175525/unable-to-install-logging-module-python

Reply:
It's just a suggestion - the error needs to be fixed before we merge it so it's worth trying